assets/base/i18n/en.json

```diff
@@ -16,6 +16,7 @@
   "about.teamEditiont1": "Enterprise Edition",
   "about.title": "About Mattermost",
   "about.version": "Version:",
+  "help.title": "Help",
```

Comment:
does this key already exists? if so no need to add it here, if it does not exists it needs to have `mobile.` prefix like `mobile.help.title`